config/production.json

```diff
@@ -18,6 +18,7 @@
 	"features": {
 		"ad-tracking": true,
 		"bilmur-script": true,
+		"build/sites-dashboard": true,
```

Comment:
Interesting! When discussing with the team I had argued that we _shouldn't_ enable it on staging first. My rationale is that doing your post-merge, pre-deploy checking on staging only works on the assumption that staging is almost identical to prod.

Reply:
A12s are great dog fooders, and many of us use the product daily, so what better way to test this without any risks? 😉 This isn't a requirement, though, just a suggestion of a process that we've previously been finding helpful to avoid breakage or flows/problems we haven't thought of while preparing to release a project.